packages/itwin/one-click-lca-widget/src/widgets/OneClickLCAWidget.tsx

```diff
@@ -12,11 +12,14 @@ import {
   StageUsage,
 } from "@itwin/appui-react";
 import OneClickLCA from "../components/OneClickLCA";
+import type { ReportProps } from "../components/Reports";
 import React from "react";
 
 export class OneClickLCAProvider implements UiItemsProvider {
   public readonly id = "OneClickLCAProvider";
 
+  constructor(private readonly _props: ReportProps) {}
```

Comment:
Made the `_props` optional. Not providing it should default to using production environment (current implementation).
Making it mandatory would result in this PR containing a breaking change and it could make it harder to upgrade for consumers.